frontend/src/hooks/useChat.tsx

```diff
@@ -50,7 +50,7 @@ const useChat = ({ sessionRef }: UseChatProps): UseChat => {
         const { text, participantName } = JSON.parse(data);
         const message: ChatMessageType = {
           timestamp: Date.now(),
-          participantName: `${participantName || 'unknown user'}`,
+          participantName: `${participantName ?? 'unknown user'}`,
```

Comment:
How come? This now accepts empty strings

Reply:
ah you're right. I followed SonarCloud's suggestions but I shouldn't have. I'll revert! thanks 🙏